src/test/scala/edu/ie3/powerFactory2psdm/common/ConverterTestData.scala

```diff
@@ -688,8 +688,8 @@ object ConverterTestData extends LazyLogging {
     "someSwitch" -> ConversionPair(
       Switch(
         "someSwitch",
-        getNodePair("someNode").input.id,
-        getNodePair("someSlackNode").input.id,
+        getNodePair("someNode").input.unsafeId,
```

Comment:
Maybe indicate within documentation of `Switch` that you need the unsafe id here?